.travis.yml

```diff
@@ -0,0 +1,34 @@
+install:
+ - mvn -s .tarvis-settings.xml clean install
```

Comment:
Is this mvn clean install really needed? Why not simply running the mvn integration-test below in the 'script' phase ?